src/redux/store.ts

```diff
@@ -10,6 +10,11 @@ export const initState: AppState = {
     loading: false,
     error: null,
   },
+  jobSeeker: {
+    credentials: '',
```

Comment:
Change the shape of `credential` for both `employer` and `jobSeeker`, because `credential` looks like:
`credentials: { email: abc, password: abc }`

Reply:
Yeah, why did you change it to be an empty string, actually?